ensime_shared/client.py

```diff
@@ -421,6 +429,15 @@ def doc_uri(self, args, range=None):
         self.log.debug('doc_uri: in')
         self.send_at_position("DocUri", False, "point")
 
+    def usages(self, args, range=None):
```

Comment:
I don't consider things "unrelated changes" though when we're talking about newly-added code. If it's impossible to define this without the unused `args` and `range` params without changing other things to make it work, then okay that's fine. But if it is possible already (that's my question, I didn't try yet), what's the justification to blindly keep adding new code this way?